src/locale/locale-en.json

```diff
@@ -540,13 +540,16 @@
         "ENGLISH": "english",
         "LIVE_SESSION_REQUEST": "Live Session Request",
         "WANT_TO_JOIN": "Join the live session.",
```

Comment:
Change text to:
"[Teacher Name] wants to start your lesson.
Please click Accept to start".